vita3k/renderer/include/renderer/vulkan/pipeline_cache.h

```diff
@@ -19,6 +19,7 @@
 
 #include <array>
 #include <map>
+#include <numeric>
```

Comment:
I believe it should be 
```suggestion
#include <limits>
```
instead

Reply:
what about the TODO below it?